bikeshed/MetadataManager.py

```diff
@@ -744,6 +745,7 @@ def joinDdList(a,b):
     "Repository": Metadata("Repository", "repository", joinValue, parseRepository),
     "Revision": Metadata("Revision", "level", joinValue, parseLevel),
     "Shortname": Metadata("Shortname", "shortname", joinValue, parseLiteral),
+    "Show Hide Deleted Button": Metadata("Show 'Hide deleted text' button", "showHideDeletedButton", joinValue, parseBoolean),
```

Comment:
That's what I have right now in my paper. No particular love for it versus anything else. Suggestion? "Hide deleted text"?